docs/testing.md

```diff
@@ -26,9 +27,12 @@ describe("My Probot app", () => {
     probot = new Probot({
       githubToken: "test",
       // Disable throttling & retrying requests for easier testing
-      Octokit: ProbotOctokit.defaults({
-        retry: { enabled: false },
-        throttle: { enabled: false },
+      Octokit: ProbotOctokit.defaults((instanceOptions:OctokitOptions):OctokitOptions =>{
```

Comment:
Do we need the explicit typings here?

Reply:
We can get rid of the strong/explicit typing and simply use `any`, this would also enable us to remove the import statement.